configs/devServerProxy.config.js

```diff
@@ -5,7 +5,7 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-const baseUrl = process.env.BASE_URL ?? 'https://qa.harness.io/gateway'
+const baseUrl = process.env.BASE_URL ?? 'https://app.harness.io/gateway'
```

Comment:
Is this intentional?

Reply:
We don't want to default to trying to use a live production system.
```suggestion
const baseUrl = process.env.BASE_URL ?? 'https://qa.harness.io/gateway'
```

Reply:
It was added by mistake. removed it now